.yarn/patches/@rive-app-canvas-npm-2.31.5-df519c6e0f.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/rive.js b/rive.js
+index d97ea4285f3d07bb47e10a68b4d10eaa81d7a0d9..9e54692d8034e25c8952693a334aaabe636bb80c 100644
+--- a/rive.js
++++ b/rive.js
+@@ -4190,7 +4190,7 @@ var __awaiter = (undefined && undefined.__awaiter) || function (thisArg, _argume
+ };
+ var __generator = (undefined && undefined.__generator) || function (thisArg, body) {
+     var _ = { label: 0, sent: function() { if (t[0] & 1) throw t[1]; return t[1]; }, trys: [], ops: [] }, f, y, t, g = Object.create((typeof Iterator === "function" ? Iterator : Object).prototype);
+-    return g.next = verb(0), g["throw"] = verb(1), g["return"] = verb(2), typeof Symbol === "function" && (g[Symbol.iterator] = function() { return this; }), g;
++    return g.next = verb(0), g["throw"] = verb(1), g["return"] = verb(2), typeof Symbol === "function" && (Object.defineProperty(g, Symbol.iterator, { value: function() { return this; } })), g;
```

Comment:
It turns out this was a gap in overrideTaming coverage of intrinsics and should be supported upstream.

The patch can be removed when https://github.com/endojs/endo/pull/2993 ships.